server/repository/src/migrations/v2_06_00/add_name_insurance_join.rs

```diff
@@ -0,0 +1,56 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_name_insurance_join"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        let policy_type = if cfg!(feature = "postgres") {
+            "insurance_policy_type"
+        } else {
+            "TEXT"
+        };
+
+        if cfg!(feature = "postgres") {
+            // Postgres changelog variant
+            sql!(
+                connection,
+                r#"
+                    ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'name_insurance_join';
+                "#
+            )?;
+
+            // Postgres enum variant for policy_type
+            sql!(
+                connection,
+                r#"
+                    CREATE TYPE insurance_policy_type AS ENUM ('Personal', 'Business');
+                "#
+            )?;
+        }
+
+        sql!(
+            connection,
+            r#"
+                CREATE TABLE name_insurance_join (
+                    id TEXT NOT NULL PRIMARY KEY,
+                    name_link_id TEXT NOT NULL REFERENCES name_link(id),
+                    insurance_provider_id TEXT NOT NULL REFERENCES insurance_provider(id),
+                    policy_number_person TEXT,
+                    policy_number_family TEXT,
+                    policy_number TEXT NOT NULL,
```

Comment:
As discussed, probably want to make "person" and "family" just boolean flags in OMS, and just translate them to OG?

Reply:
I've checked in mSupply, it looks like if both fields are filled in, they're combined in to the `full` field.
E.g. 123-321

So I think we probably want to keep it how it is...